content/en/database_monitoring/setup_oracle/troubleshooting.md

```diff
@@ -36,6 +36,17 @@ Ensure that the [recommended Agent version][2] for your hosting type is installe
 ### "Table or view does not exist" error in `agent.log`
 Execute the permission grants listed in the **Grant permissions** step of the [setup instructions][3] for your hosting type.
 
+## Known issues
+
+### No Oracle DB hostname reported
+
+Datadog Agent detects the Oracle DB hostname by running SQL against [V$INSTANCE][4].
+When Oracle DB returns `null` for the `HOST_NAME` column, the Datadog Agent reports the Oracle DB host name as empty. We have confirmed this behavior with Oracle Autonomous Database.
```

Comment:
```suggestion
When an Oracle DB returns `null` for the `HOST_NAME` column, the Datadog Agent reports the Oracle DB host name as empty. This behavior has been confirmed with Oracle Autonomous Database as well.
```

Reply:
Shall we leave `as well` out? We have confirmed the behaviour *only* with Oracle Autonomous Database.